CMP/CmpWap/CmpWapExtension/Api/Controllers/VmsController.cs

```diff
@@ -143,10 +143,10 @@ public IQueryable<CreateVm> ListVms(string subscriptionId)
                 }
 
                 var shares = from share in VmList.ToList()
-                             where string.Equals(share.SubscriptionId, 
```

Comment:
It might be a good idea to use string.IsNullOrEmpty just to cover the string.empty case as well.